pkg/proxy/apis/config/v1alpha1/defaults.go

```diff
@@ -109,6 +109,19 @@ func SetDefaults_KubeProxyConfiguration(obj *kubeproxyconfigv1alpha1.KubeProxyCo
 		// typical server timeouts.
 		obj.Conntrack.TCPCloseWaitTimeout = &metav1.Duration{Duration: 1 * time.Hour}
 	}
+
+	if obj.Conntrack.TCPBeLiberal == nil {
+		obj.Conntrack.TCPBeLiberal = pointer.Int32(-1)
+	}
+
+	if obj.Conntrack.UDPTimeout == nil {
+		obj.Conntrack.UDPTimeout = &metav1.Duration{Duration: 0 * time.Second}
+	}
+
+	if obj.Conntrack.UDPStreamTimeout == nil {
+		obj.Conntrack.UDPStreamTimeout = &metav1.Duration{Duration: 0 * time.Second}
+	}
```

Comment:
why defaulting, if is not set we should not do anything ... if you default it you have to apply that value

Reply:
I was not ware of that, will remove defaulting from here.